package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@safe-global/safe-deployments",
-  "version": "1.35.0",
+  "version": "1.35.1",
```

And this one as well, as we have separate processes for managing the package version and release cycle:

```suggestion
  "version": "1.35.0",
```

---

done